app/Http/Controllers/ReservationController.php

```diff
@@ -42,6 +55,19 @@ public function showCompletionReservation()
 
         return view('users.reservation.completion')
             ->with('confirmedReservations', $confirmedReservations)
-            ->with ('userAttribute', $userAttribute);
+            ->with('userAttribute', $userAttribute);
     }
+
+
+    public function pdf()
+    {
+        if (Auth::check()) {
+            $reservations = Reservation::with('user')->get();
```

Comment:
Are you gonna pass the $reservations in the view ? if not, consider removing it

Reply:
I've removed it